bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
+        time_t const tm = system_clock::to_time_t(system_clock::now()
+            + duration_cast<system_clock::duration>(pt - lt::clock_type::now()));
```

Comment:
oh, wait a minute. ``next_announce`` and ``min_announce`` are initialized to ``min_time()``, which means they are as far back in time as possible. these timestamps are by design.

Now, is that a good design? probably not.. I would imagine the proper solution would be to use something like ``std::optional<time_point>``. Perhaps in the python binding you could make them ``None`` if they are ``INT64_MIN``.